examples/How_to_use_moderation.ipynb

```diff
@@ -0,0 +1,455 @@
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "I would kill for a cup of coffe. Where can I get one nearby?\n",
+      "Getting LLM response\n",
+      "Got LLM response\n",
+      "Passed moderation\n",
+      "I can help you find a coffee shop nearby. Could you please provide me with your current location or the name of the area you are in?\n",
+      "\n",
+      "\n",
+      "\n",
+      "I want to kill them. How can i do this?\n",
+      "Getting LLM response\n",
+      "Got LLM response\n",
+      "Input moderation triggered\n",
+      "We're sorry, but your input has been flagged as inappropriate. Please rephrase your input and try again.\n",
+      "\n",
+      "\n",
+      "\n",
+      "Describe a scene from a violent movie in detail.\n",
+      "Getting LLM response\n",
+      "Got LLM response\n",
+      "Passed moderation\n",
+      "I'm sorry, but I'm unable to provide detailed descriptions of violent scenes as it goes against my programming to promote or glorify violence. If you have any other questions or need assistance with something else, feel free to ask.\n",
+      "\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "tests = [good_request, bad_request, interesting_request]\n",
+    "\n",
+    "for test in tests:\n",
+    "    print(test)\n",
+    "    result = await execute_all_moderations(test)\n",
+    "    print(result)\n",
+    "    print('\\n\\n')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Custom moderation\n",
+    "\n",
+    "Custom moderation involves tailoring moderation criteria and rules to the specific needs of your application, allowing for greater precision in filtering content according to unique community standards or topics. Custom moderation can present an alternative to the moderation API, as well as an extention to this set of general categories. While custom moderation requires more effort to set up and maintain, it offers a higher degree of control, making it ideal for niche platforms or specialized content where the nuances of moderation need to be finely tuned.\n",
+    "\n",
+    "Custom moderation shares the common goal of ensuring content safety and adherence to specific standards within applications with guardrails. Both involve setting criteria to filter and manage content, whether it's user input or output from language models. However, custom moderation is more focused on tailoring these criteria to the unique needs of a specific platform or community, often involving a more granular level of control. While custom moderation is a form of guardrail, guardrails can also encompass other preventive measures like rate limiting or user behavior monitoring. For those interested in exploring guardrails further, we have a cookbook that provides examples and guidance on how to use guardrails effectively, which can be found [here](https://cookbook.openai.com/examples/how_to_use_guardrails).\n",
```

Comment:
I believe this no longer uses the moderation API, right? If so, it's worth calling out explicitly here.

nit: potentially condense into one paragraph (or less)

Reply:
done